File: PHP/level-100.php
<?php
    $a = 0;
    $b = 0;
    $c = 0;
    $d = 0;

    for($a = 1; $a <= 100; $a++) {
        $b = $a % 3;
        $c = $a % 5;
        $d = $a % 15;

        if ($d == 0) {
            echo "FizzBuzz", "\n";
            continue;
        }

        if ($b == 0) {
            echo "Fizz", "\n";
            continue;
        }

        if ($c == 0) {
            echo "Buzz", "\n";
            continue;
        }

        echo $a, "\n";

    }
?>

File: Java/level-100.java
public class Main {
    public static void main(String[] args) {
        int b = 0;
        int c = 0;
        int d = 0;

        for (int a=1; a<=100; a++) {
            b = a % 3;
            c = a % 5;
            d = a % 15;

            if (d == 0) {
                System.out.println("FizzBuzz");
                continue;
            }
            if (b == 0) {
                System.out.println("Fizz");
                continue;
            }
            if (c == 0) {
                System.out.println("Buzz");
                continue;
            }
            System.out.println(a);
        }
    }
}
